number-of-TiDB-contributors-exceeded-400.md

```diff
@@ -0,0 +1,48 @@
+---
+title: TiDB Contributor 人数突破 400，有关开源理想，我们同在！
+author: ['PingCAP']
+date: 2020-03-04
+summary: 和一群志同道合的朋友一起做酷且正确的事情，哪怕它是困难的、甚至曾被人认为是不可能的。但，让世界变得更美好，不正是我们踏入开源世界的初衷吗？
+tags: ['社区','社区动态']
+---
+从 2015 年 5 月写下第一行代码，TiDB 作为一款新型开源分布式数据库项目，走过了 0 到 1 的**蜕变之日**；走过了存储层 TiKV 项目晋级为 CNCF Incubating Projects 的**光辉时刻**；周边工具逐步完善，**羽翼渐丰**；逐渐走向**成熟稳健**，在近 1000 家用户的生产环境中不断打磨升级。
+
+**这一切荣誉和成就都属于社区——**
+
+**TiDB Repo Contributor 突破 400,**  
+**Thanks for your contributions!**
+
+![](media/Number-of-TiDB-contributors-exceeded-400/1-400-contributor.png)
+
+**近一年 TiDB 开源社区架构和治理规则也在不断规范和升级迭代中：**
+
+* 社区自治组织和角色：
+	* 目前为止，社区启动过 20 个 Working Group (WG) ， 成立了 5 个 Special Interest Group（SIG），它们都有了丰富的阶段性产出；
+	* 除了已有的 Maintainer、Committer、Contributor 角色之外， 新增了 Reviewer、Active Contributor 角色，让大家找到了“一步步升级打怪”的乐趣。
+
+* 社区活动愈加丰富：
+	* 除了传统的 Infra Meetup、TiDB TechDay、TiDB DevCon 线下交流活动，颇具 Geek 精神的 TiDB Hackathon 已成功举办两届，诞生了很多奇思妙想的项目；
+	* TiDB 性能挑战赛第一季 落幕，第二季易用性挑战赛 正在进行中，接下来的 3 个月参赛选手将挑战“让 TiDB 变得更好用”；
+	* 本周末即将举办 Book Rush 活动，社区伙伴们挑战 48 小时合写一本 5～6 万字的 TiDB 技术书籍。
```

Comment:
Book Rush：https://pingcap.com/blog-cn/blog-cn/tidb-book-rush-write-a-book-in-48-hours

Reply:
done